dashboards/magento-business-insights/magentoBI.json

```diff
@@ -0,0 +1,4892 @@
+        {
+          "title": "Total Order Value",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 2,
+            "height": 1
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM MySQLSalesOrderCumulative SELECT latest(grand_total_sum) AS 'Total Order Value ($)'"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Average order value (AOV) ($)",
+          "layout": {
+            "column": 3,
+            "row": 1,
+            "width": 2,
+            "height": 1
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "from MySQLSalesOrderCumulative select latest(average_grand_total) as 'Average order value ($)'"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Orders Count",
+          "layout": {
+            "column": 5,
+            "row": 1,
+            "width": 2,
+            "height": 1
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "SELECT uniqueCount(`Order ID`) as 'Orders Count' FROM MYSQLSalesOrderByProduct since this month compare with 1 month ago"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Orders Item Count",
+          "layout": {
+            "column": 7,
+            "row": 1,
+            "width": 2,
+            "height": 1
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM MySQLSalesOrderCumulative select latest(total_order_item_count) as 'Orders Item Count' since this month COMPARE WITH  1 month ago"
```

Comment:
```suggestion
                "query": "FROM MySQLSalesOrderCumulative select latest(total_order_item_count) as 'Ordered Items Count' since this month COMPARE WITH  1 month ago"
```